src/src/com/microsoft/aad/adal/AuthenticationSettings.java

```diff
@@ -44,7 +44,7 @@
      */
     private String mSharedPrefPackageName;
 
-    private boolean mSkipBroker = false;
+    private boolean mUseBroker = true;
```

Comment:
my first thought on this is, if i want to use broker i'll call setUseBroker, and it will be true. 
I think the default should be false.